guides/common/modules/con_introduction-to-organization-and-location-context.adoc

```diff
@@ -0,0 +1,12 @@
+[id="Introduction_to_Organization_and_Location_Context_{context}"]
+= Introduction to Organization and Location Context in {Project}
+
+Context in {Project} consists of xref:Managing_Organizations_{context}[Organizations] and xref:Managing_Locations_{context}[Locations].
+
+Most entities in {Project} are associated with at least one organization and location context.
```

Comment:
Ok, now this is going to be the hardest part and we'll probably need some help getting this right. 
I grepped our guides for the term "entity" and it doesn't exist (it does in SOE guide, which isn't straight docs)
This is the first time we use this term... It should probably be defined somewhere first, as it means something and nothing at the same time.. Does entities = resources here? 
Could we use a definition list to give examples of what we mean by entity and what we mean by (resource) if this is different? I understand if you're not able to do this off the top of your head. Perhaps @ezr-ondrej can help us get right since he has spent a lot of time thinking about it.

Reply:
I think entity = resource, but I can only see Rails model behind both, so it is hard for me to imagine what would user understand by those respectively :thinking:

Reply:
Replaced `entities` with `resources`.